devops/kustomize/base/api/deploy.yaml

```diff
@@ -59,11 +59,11 @@ spec:
               protocol: TCP
           resources:
             limits:
-              cpu: 200m
-              memory: 1Gi
+              cpu: 100m
+              memory: 1500Mi
```

Comment:
The API was running out of memory.  We can't go any higher however because our namespace has run out of memory.